pages/index.tsx

```diff
@@ -73,7 +93,7 @@ export default function Home({
 
       const createdGift = await createGift(newGift);
       const updatedGiftList = giftData.concat(createdGift);
```

Comment:
Uuden lahjan luonnin jälkeen meidän pitää kertoa react querylle, että lahjadata ei ole enää ajantasalla, jolloin se päivitetään automaattisesti. Tämä kertominen tehdään [invalidoimalla query](https://tanstack.com/query/latest/docs/framework/react/guides/query-invalidation).
```suggestion
// Tämä tulee `Home` componentin alkuun
const queryClient = useQueryClient()

// Tämä tulee tähän kohtaan
queryClient.invalidateQueries({ queryKey: ['gifts'] })
```